remme/clients/block_info.py

```diff
@@ -46,6 +46,7 @@ def get_many_block_info(self, start, limit):
     def get_block_info_config(self):
         bic = BlockInfoConfig()
         bic.ParseFromString(self.get_value(CONFIG_ADDRESS))
+        LOGGER.info(f'get_block_info_config: {bic}')
```

Comment:
What is this log entry for? Should it be in INFO level? Make an understandable log entry.

Reply:
Removed